### DISCUSSION TITLE
使用预设补帧方案会将之前的滤镜清除怎么办

### DISCUSSION THREAD

**Discussion by OrangeYoon** [hooke007/mpv_PlayKit#463] [mpv 通用]
有没有砖家有解决办法的，谢谢。

**Comment by hooke007** [comment]
看看说明书
https://github.com/hooke007/MPV_lazy/blob/2027fb8b2ec766896773c6803c9b7a33a4fc6f12/portable_config/input_list.conf#L44

**Reply by OrangeYoon** [reply]
<img width="746" alt="屏幕截图 2024-05-21 141201" src="https://github.com/hooke007/MPV_lazy/assets/169504647/73cb910e-54b8-4827-8ef6-93a442a70cce">
请问这个报错如何解决，使用命令也会出现，是哪里不对呢

**Reply by OrangeYoon** [reply]
按钮切换也会这样。脚本不知道哪里在初始化的时候请求帧了

**Reply by hooke007** [reply]
哪里有报错？？？
只有红色是报错
没有东西需要解决

**Comment by OrangeYoon** [comment]
老哥辛苦了，亲自回复，

Karl Hook ***@***.***>于2024年5月21日 周二18:27写道：
